Code-ologists/README.md

```diff
@@ -2,9 +2,48 @@
 PyDSA Visualization Examples
 
 **Team**: Code-ologists
-<br>
-**Team Members**: Kanika Murarka ( _@kanikaa1234 )
-		  Anika Murarka (_@anikamurarka )
-<br>
-**Algorithm Implemented**: Bubble Sort
- 
+
+**Team Members**: Kanika Murarka [kanikaa1234](https://github.com/kanikaa1234)
+		  		    Anika Murarka [anikamurarka](https://github.com/anikamurarka)
+
+**Set up environment**
+
+Create and activate virtual environment.
+```bash
+$ pip install -r requirements.txt
+```
+
+**Set-up environment variables**
+After activating your virtual environment export the following variables:
+
+```bash
+export SECRET_KEY="alskj2mn34m@$%asd#45ASD"
```

Comment:
You stored keys in env variables, but if you commit them here, then the whole concept of having this in the env variables looks redundant.

It's better to show an example .env file in README.

Although this would work for this PR. But remember that you never commit keys anywhere as discussed.